name.abuchen.portfolio.junit/src/name/abuchen/portfolio/junit/TestCurrencyConverter.java

```diff
@@ -55,31 +76,24 @@ public String getTermCurrency()
         return termCurrency;
     }
 
-    @Override
-    public Money convert(LocalDate date, Money amount)
-    {
-        try
-        {
-            return CurrencyConverter.super.convert(date, amount);
-        }
-        catch (MonetaryException e)
-        {
-            // testing: any other currency will be converted 1:1
-            if (!amount.getCurrencyCode().equals(series.getBaseCurrency()))
-                return Money.of(termCurrency, amount.getAmount());
```

Comment:
Just to point it out, this is where the 1:1 conversion happens in the old code …